chalicelib/checks/es_checks.py

```diff
@@ -91,4 +94,4 @@ def clean_s3_es_checks(connection, **kwargs):
     full_output['n_deleted_es'] = n_deleted_es
     check.status = 'DONE'
     check.full_output = full_output
-    return action
+    return check
```

Comment:
So you're returning a different value. What was `action` coming from before and why wasn't it needed? I don't even see it being assigned.

Reply:
I don't know where `action` came from, I noticed it while looking at this file and fixed it. I presume it meant to be `check` always.